src/styles/card-styles.css

```diff
@@ -0,0 +1,98 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  --text-dark-color: #060b35;
+  --text-light-color: #616070;
+  --button-blue-color: #00acdc;
```

Comment:
It's better to define variables in the [:root](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) selector